db/structure.sql

```diff
@@ -165,7 +165,7 @@ CREATE TABLE public.offices (
     street text,
     city text,
     postcode text,
-    location public.geometry(Point),
+    location public.geography(Point,4326),
```

Comment:
I had to look up 4326. It just seems like such an arbitrary way to make the earth spherical! 🌍